test/geocoders/geocodefarm.py

```diff
@@ -96,6 +96,35 @@ def mock_call_geocoder(*args, **kwargs):
         with self.assertRaises(exc.GeocoderQuotaExceeded):
             self.geocoder.geocode('435 north michigan ave, chicago il 60611')
 
+    def test_no_results(self):
+        """
+        GeocodeFarm unhandled error
```

Comment:
This is clearly a copy-paste from another test. You can remove this comment completely (instead of updating it), because after switch to pytest (#266) they all will be removed anyway.

Reply:
Yes, I wanted to emulate existing tests instead of reinventing unnecessarily... I was sloppy and missed that comment. Will remove.

Reply:
@KostyaEsmukov removed that comment